data-source/jsonnet/northern-ireland/census_individual.jsonnet

```diff
@@ -151,6 +155,14 @@ function(region_code) {
             sexual_identity,
           ],
         },
+        {
+          id: 'school-group',
+          title: 'School',
+          blocks: [
+            school_location,
+            school_travel_mode,
+          ],
+        },
```

Comment:
This group seems a bit unnecessary, maybe we could rename `Qualifications` to `Education` and merge these blocks into that group?

Reply:
As it is reached from multiple groups, and we can't deep link into groups, this seems necessary.